cfme/fixtures/configure_auth_mode.py

```diff
@@ -67,24 +70,34 @@ def configure_aws_iam_auth_mode(browser, available_auth_modes):
         yield
 
 
-def _configure_external_auth_ipa(request):
-    if "ipa" not in cfme_data:
-        pytest.skip("No IPA server in configuration!")
-    data = cfme_data.get("ipa", {})
-    request.addfinalizer(disable_external_auth_ipa)
-    setup_external_auth_ipa(**data)
-
-
-@pytest.fixture(scope="function")
-def configure_external_auth_ipa(request):
-    _configure_external_auth_ipa(request)
-
-
-@pytest.fixture(scope="module")
-def configure_external_auth_ipa_module(request):
-    _configure_external_auth_ipa(request)
-
-
-@pytest.fixture(scope="class")
-def configure_external_auth_ipa_class(request):
-    _configure_external_auth_ipa(request)
+@pytest.fixture()
+def configure_auth(request, auth_mode):
+    data = cfme_data['auth_modes'].get(auth_mode, {})
+    if version.appliance_is_downstream() and request.node.name in [
+        'test_auth_configure[ext_ipa-retrieve_group]',
+        'test_auth_configure[ext_openldap-create_group]',
+        'test_auth_configure[ext_openldap-retrieve_group]',
+        'test_auth_configure[ext_openldap-evm_default_group]'
+    ]:
+        pytest.skip("skip tests in downstream due to these known open issues:"
+                    "\nhttps://bugzilla.redhat.com/show_bug.cgi?id=1394066"
+                    "\nhttps://bugzilla.redhat.com/show_bug.cgi?id=1394425")
```

Comment:
@amavinag, this is not how we skip BZs. You should use `@pytest.mark.meta(blockers=[...ids...])` instead, potentially with `forced_stream=['5.6', '5.7']` if it's not skipping properly.